validator/app/src/compute_horde_validator/validator/dynamic_config.py

```diff
@@ -31,6 +31,24 @@ async def aget_config(key):
     return await dynamic_config_holder.get(key)
 
 
+async def aget_number_of_prompts_in_series():
+    if settings.DEBUG_OVERRIDE_DYNAMIC_NUMBER_OF_PROMPTS_IN_SERIES is not None:
+        return settings.DEBUG_OVERRIDE_DYNAMIC_NUMBER_OF_PROMPTS_IN_SERIES
+    return await aget_config("DYNAMIC_NUMBER_OF_PROMPTS_IN_SERIES")
+
+
+async def aget_number_of_prompts_to_validate_from_series():
+    if settings.DEBUG_OVERRIDE_DYNAMIC_NUMBER_OF_PROMPTS_TO_VALIDATE_FROM_SERIES is not None:
+        return settings.DEBUG_OVERRIDE_DYNAMIC_NUMBER_OF_PROMPTS_TO_VALIDATE_FROM_SERIES
+    return await aget_config("DYNAMIC_NUMBER_OF_PROMPTS_TO_VALIDATE_FROM_SERIES")
```

Comment:
These functions seem to be only used in sync context and not in a hot-loop. So sync-to-async-to-sync conversion seems overkill. (same for the other two)

```suggestion
def get_number_of_prompts_to_validate_from_series():
    if settings.DEBUG_OVERRIDE_DYNAMIC_NUMBER_OF_PROMPTS_TO_VALIDATE_FROM_SERIES is not None:
        return settings.DEBUG_OVERRIDE_DYNAMIC_NUMBER_OF_PROMPTS_TO_VALIDATE_FROM_SERIES
    return config.DYNAMIC_NUMBER_OF_PROMPTS_TO_VALIDATE_FROM_SERIES
```

Reply:
If `DEBUG_` settings are not strictly needed (as you can easily override these in tests), I'd remove these functions altogether and use `config.` directly at the call-sites.